tcms/testcases/tests/test_models.py

```diff
@@ -157,3 +159,28 @@ def test_send_mail_to_case_author(self, send_mail):
             recipients,
             fail_silently=False,
         )
+
+
+class TestActualDurationProperty(TestCase):
+    """Test TestExecution.actual_duration"""
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.test_execution = TestExecution()
+        cls.test_execution.start_date = timezone.now()
+        cls.test_execution.stop_date = timezone.now() + timezone.timedelta(days=1, seconds=0, microseconds=0)
+
+    def test_calculation_of_actual_duration(self):
+        self.assertEqual(
+            self.test_execution.actual_duration, timezone.timedelta(days=1, seconds=0, microseconds=2)
```

Comment:
This is unreliable b/c on different executions in CI it can take more or less than 2 microseconds between assigning the values for start_date and stop_date on llines 172, 173.

If you look at the logs from GitHub CI you will see that the resulting delta is 1 day, 4 microseconds instead of the expected 1day, 2 microseconds. 

If you prefer to use `timezone.now()` then `stop_date = start_date + hard-coded timedelta`.  Alternatively
```
start_date = datetime(<hard-coded year, month, day, hours, etc>)
stop_date = datetime(<another hard-coded value>)
```